lib.rs

```diff
@@ -280,22 +280,7 @@ where
         let join = builder.spawn(move || loop {
             match rx.recv().unwrap() {
                 AsyncMsg::Record(r) => {
-                    let rs = RecordStatic {
-                        location: &*r.location,
-                        level: r.level,
-                        tag: &r.tag,
-                    };
-
-                    drain
-                        .log(
-                            &Record::new(
-                                &rs,
-                                &format_args!("{}", r.msg),
-                                BorrowedKV(&r.kv),
-                            ),
-                            &r.logger_values,
-                        )
-                        .unwrap();
+                    r.to_drain(&drain).unwrap();
```

Comment:
```suggestion
                    r.to_log(&drain).unwrap();
```

Reply:
I'd go with `log_to` because it logs it to a logger/drain. `to_log` sounds like it converts it to a logger.